doc/VitessReplication.md

```diff
@@ -97,37 +96,36 @@ Also, Vitess adds comments to the rewritten statements that identify the primary
 key affected by that statement. This allows us to produce an Update Stream (see
 section below).
 
-## Vitess Pivot Schema Changes
+## Vitess Schema Swap
 
 Within YouTube, we also use a combination of statement based replication and
 backups to apply long-running schema changes without disrupting ongoing
-operations. See the [pivot tutorial](/user-guide/pivot-schema-changes.html)
+operations. See the [schema swap tutorial](/user-guide/schema-swap.html)
```

Comment:
Might as well rename and update `pivot-schema-changes.md` since you deleted its source doc.

There are also other places that link to the doc you renamed:

```
doc/ProductionPlanning.md:70:* [Schema Management](http://vitess.io/user-guide/schema-management.html) / [Pivot Schema Changes](http://vitess.io/user-guide/pivot-schema-changes.html)
doc/SchemaManagement.md:10:Note that this functionality is not recommended for long-running schema changes. In such cases, we recommend to do a [pivot schema change](/user-guide/pivot-schema-changes.html) instead.
doc/SchemaManagement.md:158:However, we do not recommend this. Instead, you should apply large schema changes by following the [pivot process](/user-guide/pivot-schema-changes.html).
vitess.io/_includes/left-nav-menu.html:29:                <li style="padding-bottom: 0"><a href="/user-guide/pivot-schema-changes.html">Pivot Schema Changes (Tutorial)</a></li>
```